openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityThreadPanel/ActivityThreadPanel.tsx

```diff
@@ -351,12 +383,25 @@ const ActivityThreadPanel: FC<ActivityThreadPanelProp> = ({
               postFeed={postFeed}
               selectedThreadId={selectedThreadId}
               threads={threads}
+              onConfirmation={onConfirmation}
               onThreadIdSelect={onThreadIdSelect}
               onThreadSelect={onThreadSelect}
             />
           </Fragment>
         )}
       </div>
+      {confirmationState.state && (
+        <ConfirmationModal
+          bodyClassName="tw-h-18"
+          bodyText="Are you sure you want to permanently remove this post?"
```

Comment:
Can we make this a component to avoid code duplication since we are using the same confirmation modal in at least 3 components?
So, that this can be replaced with 
```
{confirmationState.state && (
  <DeleteConfirmationModal onCancel={onDiscard} onConfirm={onDelete} />
)}
```